.coderabbit.yaml

```
@@ -49,20 +51,31 @@ reviews:
             * Test any remaining specifics (e.g. component-level stuff)
             * Do the work
         
         Because this is an annotation processor, tests should be included by adding modules/records to the `utils-tests` and should include positive and negative cases
-          * The design of the project may mean that the "negative" case isn't explicit, but is tested elsewhere. The positive case should definitely be tested though!
\ No newline at end of file
+          * The design of the project may mean that the "negative" case isn't explicit, but is tested elsewhere. The positive case should definitely be tested though!
+        
+        The documentation here is a little more sparse, which isn't great but the aim is that each item should mostly be self-contained or should have a relationship with a
+          small parent that helps out with the context of the work. If parent classes do exist or the class is something used in multiple places, it should be documented.
+
+        Every package must be `NullMarked`, and if code is added that can put `null` somewhere that isn't explicitly annotated as `Nullable` it should be called out.
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)